### ISSUE TITLE
Neat—but how can we respect transform origin?

### ISSUE THREAD

**Issue by nikkwong** [elrumordelaluz/svg-path-tools#1]
I like this idea, but is it possible to pass an origin for the scale center, for example?

**Comment by elrumordelaluz** [comment]
Interesting @nikkwong, thanks to point that. Let's investigate and give you a feedback asap.

**Comment by nikkwong** [comment]
Oh man, that'd be awesome. How would you approach it? For example, I guess if the origin is `bottom right`, you'd have to calculate the offset of each point from the most bottom right point, and multiply that by the scale factor. Sort of complicated.

**Comment by elrumordelaluz** [comment]
Yes, something like this. I started working with something like `bbox` to help in similar situations. Will show you here.

**Comment by ziaenezhad** [comment]
++
